sw/device/silicon_creator/manuf/lib/personalize_functest.c

```diff
@@ -114,17 +100,9 @@ bool test_main(void) {
 
     // Provision the OTP SECRET2 partition and flash info pages.
     if (!status_ok(manuf_personalize_device_secrets_check(&otp_ctrl))) {
-      // Wait for host ECC pubkey, used to generate a shared AES key to export
-      // the RMA unlock token, to arrive over the console.
-      LOG_INFO("Ready to receive host ECC pubkey ...");
-      ecc_p256_public_key_t host_ecc_pk;
-      CHECK_STATUS_OK(UJSON_WITH_CRC(ujson_deserialize_ecc_p256_public_key_t,
```

Comment:
This is where I would keep the ujson console read/deserialization of the token hash.

Reply:
done